config/dispatcher/dispatcher.ex

```diff
@@ -73,7 +73,7 @@ defmodule Dispatcher do
   end
 
   match "/organizations/*path", %{ accept: [:json], layer: :api} do
-    Proxy.forward conn, path, "http://cache/organizations/"
+    Proxy.forward conn, path, "http://resource/organizations/"
```

Comment:
As discuss: 
- [ ] revert and check if it's working after restart `resource` and `cache` services

Reply:
With restarting the cache it does seem to work as expected without this change. I removed the commit introducing this change.

Reply:
I ~~spoke~~ wrote to soon apparently. I encountered an issue with using the cache path. When you edit, for instance, the related organizations for an organization the results shown after saving are not updated and keep showing the state from before editing. Changing the path to resource to bypass the cached results resolves this issue.